prometheus_hardware_exporter/collector.py

```diff
@@ -5,13 +5,134 @@
 
 from prometheus_client.metrics_core import GaugeMetricFamily, InfoMetricFamily
 
+from .collectors.perccli import PercCLI
 from .collectors.sasircu import LSISASCollectorHelper, Sasircu
 from .collectors.storcli import StorCLI
 from .core import BlockingCollector, Payload, Specification
 
 logger = getLogger(__name__)
 
 
+class PowerEdgeRAIDCollector(BlockingCollector):
+    """Collector for PowerEdge RAID controller."""
+
+    perccli = PercCLI()
+
+    @property
+    def specifications(self) -> List[Specification]:
+        return [
+            Specification(
+                name="perccli_command_success",
+                documentation="Indicates if the command is successful or not",
+                metric_class=GaugeMetricFamily,
+            ),
+            Specification(
```

Comment:
I didn't find this one in the spec
https://docs.google.com/spreadsheets/d/1KiQ5cS36zfjf8V32RoD1RKx9q01AhgoKvBQRkonQv5o/edit#gid=1149993398